src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -293,6 +293,13 @@ public static class ConfigurationKeys
         /// </summary>
         public const string Convention = "OTEL_CONVENTION";
 
+        /// <summary>
+        /// Configuration key for the propagator logic to be used.
+        /// Default is <c>Datadog</c>.
+        /// <seealso cref="TracerSettings.Propagator"/>
+        /// </summary>
+        public const string Propagators = "OTEL_PROPAGATORS";
```

Comment:
Is there a particular reason this is the plural "Propagators" instead of the singular "Propagator"?

Reply:
@zacharycmontoya  https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/92#discussion_r600330255